src/modules/fancyzones/editor/FancyZonesEditor/MainWindow.xaml

```diff
@@ -445,6 +445,7 @@
         </Grid>
 
 		<StackPanel Orientation="Horizontal" Margin="0,10,0,16">
+            <Button x:Name="ResetButton" Content="{x:Static props:Resources.Reset_Layout}" Padding="8" Style="{StaticResource secondaryButton}" Click="ResetButton_Click"/>
```

Comment:
nit: just to align with the names of other callbacks, we can call this `Reset_Click` (or rename other two into `EditLayoutButton_Click` and `ApplyButton_Click`)